amgut/db/patches/0018.sql

```diff
@@ -0,0 +1,65 @@
+-- August 19, 2015
```

Comment:
tested on a production copy of the database, and does this mess with creating new surveys or editing existing?